# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/linux/SandboxFilter.h
# Commit: 10e01cf10faf
# Full Hash: 10e01cf10fafca4a150c2f76b3ce4e33bbd06223
# Author: Michael Froman <mfroman@mozilla.com>
# Date: 2020-03-09 21:52:54
# Regressor Bug: 1608558
# File Overlap Count: 1
# Description:
#   Bug 1608558 - pt1 - add linux sandboxing to socket process. r=gcp
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D60014
# ==============================================================================

diff -r 61e14505a488 -r 10e01cf10faf security/sandbox/linux/SandboxFilter.h
--- a/security/sandbox/linux/SandboxFilter.h	Fri Mar 06 20:29:47 2020 +0000
+++ b/security/sandbox/linux/SandboxFilter.h	Mon Mar 09 14:56:03 2020 +0000
@@ -35,6 +35,9 @@
 UniquePtr<sandbox::bpf_dsl::Policy> GetDecoderSandboxPolicy(
     SandboxBrokerClient* aMaybeBroker);
 
+UniquePtr<sandbox::bpf_dsl::Policy> GetSocketProcessSandboxPolicy(
+    SandboxBrokerClient* aMaybeBroker);
+
 }  // namespace mozilla
 
 #endif